app/views/splash/home.html.erb

```diff
@@ -0,0 +1,15 @@
+<section id="cover" class="min-vh-100 d-flex flex-column justify-content-center align-items-center">
+  <div id="cover-caption" class="w-100 position-relative">
+    <div class="container">
+      <div class="row text-dark">
+        <div class="col-xl-5 col-lg-6 col-md-8 col-sm-10 mx-auto">
+          <h1 class="display-1 text-center mt-5">ManageMyMoney</h1>
+            <form class="d-flex flex-column text-center gap-2 splash-form mb-3">
+              <%= link_to 'SIGN IN', new_user_session_path, class: "btn btn-primary btn-block" %>
+              <%= link_to 'SIGN UP', new_user_registration_path, class: "text-decoration-none text-dark fs-6" %>
+            </form>
+        </div>
+      </div>
+    </div>
+  </div>
```

Comment:
- [ ] Kindly consider matching the splash screen with the design guideline like the following one

| Your design  | Design guidline |
| ------------- | ------------- |
| ![image](https://user-images.githubusercontent.com/53402600/227624174-bf434d66-f0bf-42c1-97bf-fb06dbe8cc47.png) | ![image](https://user-images.githubusercontent.com/53402600/227624024-f29f82a3-d11a-43c8-b16c-2bcdf8329397.png) |